src/api/tests/approversMiddleware.spec.ts

```diff
@@ -9,6 +9,7 @@ describe('Approvers Service Tests', () => {
   const withToken = useTestServer();
 
   afterEach(() => {
+    jest.restoreAllMocks();
     jest.resetAllMocks();
```

Comment:
I believe I can get rid of `resetAllMocks()` because `restoreAllMocks()` does everything `resetAllMocks()` does and more.

> mockFn.mockRestore()
> Does everything that [mockFn.mockReset()](https://jestjs.io/docs/mock-function-api#mockfnmockreset) does, and also restores the original (non-mocked) implementation.

Per: https://jestjs.io/docs/mock-function-api#mockfnmockrestore

Reply:
you're right.  I removed resetAllMocks()